Cmdr/BuiltInTypes/Player.lua

```diff
@@ -23,6 +23,13 @@ local playerType = {
 	Default = function(player)
 		return player.Name
 	end;
+
+	ArgumentOperatorAliases = {
```

Comment:
Should these (specifically all and others) be included within the singular Player type?

Reply:
Would make it easier for set up. Otherwise you'd have to edit the Util module, since MakeListableType just sets the ArgumentOperatorAliases on the listable version of the type.

When it is in singular context the all and others keywords arent used by default due to functionality in Argument:Transform